client/app/shared/text-box/text-box.component.ts

```diff
@@ -10,12 +10,13 @@ import { itemSchema, itemDataModel } from '../UIConfig.model';
 export class TextBoxComponent extends BaseComponent implements OnInit {
   labelConfig: itemSchema = new itemSchema();
   itemDataModel;
-  modelRef;
+  modelRef = { value: "" };
 
   ngOnInit() {
     this.itemDataModel = itemDataModel;
```

Comment:
Need to remove the if check as there will be no case when we won't have the model reference for text box component.